LoopKitUI/CGMManagerUI.swift

```diff
@@ -9,15 +9,20 @@ import LoopKit
 import HealthKit
 
 
-public protocol CGMManagerUI: CGMManager {
+public protocol DeviceManagerUI {
+    // An image representing the device configuration
+    static var smallImage: UIImage? { get }
+    static var name: String { get }
+    static var details: String { get }
+}
+
+public protocol CGMManagerUI: CGMManager, DeviceManagerUI {
     /// Provides a view controller for setting up and configuring the manager if needed.
     ///
     /// If this method returns nil, it's expected that `init?(rawState: [:])` creates a non-nil manager
     static func setupViewController() -> (UIViewController & CGMManagerSetupViewController & CompletionNotifying)?
 
     func settingsViewController(for glucoseUnit: HKUnit) -> (UIViewController & CompletionNotifying)
-
-    var smallImage: UIImage? { get }
```

Comment:
DIY relies on this property being an instance variable to vend the correctly-colored variant of a Minimed pump.

Reply:
Ah, good to know.  I'll add an extension to support that